src/lib.rs

```diff
@@ -12,6 +12,8 @@ pub use {array::Array, single::Single};
 
 pub use {error::Error, mapper::Mapper};
 
+pub use memoffset;
```

Comment:
Is this necessary? Are users required to use things of `memoffset`?

Reply:
Not directly, but by macro expansion.

`#[derive(BoundSetGenericOf)]` for `struct Foo { x: u32, y: u32 }` will expand to something like this:
```
impl<M,A> BoundSetGeneric<Foo, M, A> for array::Generic<Foo, M, A>
where M: Mapper, A: AccessorTypeSpecifier + 'static {
    type BoundSetGenericType<'a> = BoundSetGenericOfFoo<'a, M, A> where Self: 'a;
    fn set_at<'a>(&'a self, i: usize) -> BoundSetGenericOfFoo<'a, M, A>{
        assert!(i < self.len());
        unsafe{
            let addr = self.addr(i);
            BoundSetGenericOfFoo {
                x: single::Generic::new(addr + memoffset::offset_of!(Foo, x), Identity),
                y: single::Generic::new(addr + memoffset::offset_of!(Foo, y), Identity),
                _lifetime: PhantomData
            }
        }
    }
}
```
You see that `memoffset::offset_of` macro is used to calculate offsets of each fields. That's what I meant to say in

> Crate `memoffset` is re-exported as `accessor::memoffset` so that macro expansion can resolve it corretly.
> I'm new to proc-macros, so I'm not sure if it's a right way to refer 3rd-party crates in proc-macro code generation.

For the same reason I exposed `a.addr(i)` as public, although it won't be shown in documentation or IDE autocomplete.

Reply:
OK, thanks. After some googling, I found `pub use` seemed a common way for this purpose.